### PR TITLE
Run tests on Circle CI

### PR DESCRIPTION
**Pull request by abesto**
The latest run is https://circleci.com/gh/abesto/zipkin-ui/7

Points of interest:

 * Correct version of NPM installed before installing dependencies
 * CircleCI knows about test cases (via JUnit reporting)
 * CircleCI "knows" about coverage, by copying the generated HTML coverage reports into the artifacts directory
 * Integrates with Coveralls for coverage reporting (will need to set Coveralls API token on the CircleCI UI)
 * Currently fails with `Error: No provider for ZipkinService!`, assuming a Zipkin service is missing from the test env, or else some mocking is not working correctly.

### CODE REVIEW

**Comment by abesto** [comment]
CC @rogeralsing @adriancole

**Comment by adriancole** [comment]
added a body to the failing spec. fingers crossed

**Comment by adriancole** [comment]
looks good @abesto. how about we add a commit to remove travis config, then merge in whatever commit count you see fit?

File: karma.conf.js [hunk 1]
```diff
@@ -26,7 +26,14 @@ module.exports = function (config) {
             ]
         },
 
-        reporters: [ 'mocha', 'coverage' ],
+        junitReporter: {
+            outputDir: 'junit/',
+            useBrowserName: false
+        },
+
+        reporters: [ 'mocha', 'coverage', 'junit', 'coveralls' ],
```
Comment (adriancole) [inline karma.conf.js]: ps we should probably copy the comment on "why junit" here, since there's no java code and it might confuse ppl

**Comment by abesto** [comment]
@adriancole Added comment, squashed. There's no Travis config in the repo, the tests can be shut down via the Travis UI.

🎵 If you're feeling happy merge

**Comment by adriancole** [comment]
Thx flipped off travis too

**Pull request merged**